hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -661,7 +661,7 @@
             - Verify that we will get STALL response when try to attempt IN/OUT transactions.
             '''
       stage: V2
-      tests: []
+      tests: [usbdev_stall_trans]
```

Comment:
This is missing some quotes, which is causing CI failures. Other testpoints have what you need (e.g. `pkt_received`)